### PR TITLE
add UA locale

### PR DESCRIPTION
**Pull request by daniloff200**
Related to #25 

@eldadfux  could you please add hacktoberfest label to this PR? :)

### CODE REVIEW

**Comment by eldadfux** [comment]
(copy->paste from the RU pull request) -> Hey @daniloff200 thank you so much, amazing work and a great contribution to our project!

Would you like me to wait with the merge for 1st of October for it to be eligible as a DigitalOcean PR? (or maybe you have to submit it after 1st of October - not really sure).

Anyway contact me on our Gitter community to get some Appwrite SWAG 🥳.

**Comment by Kolkies** [comment]
> Would you like me to wait with the merge for 1st of October for it to be eligible as a DigitalOcean PR? (or maybe you have to submit it after 1st of October - not really sure).

You have to create the pr after the 1st of October, if I understand the rules

**Comment by daniloff200** [comment]
Yeah, looks like you're right @Kolkies 
anyway, there's a whole month, so, that's okay

**Comment by Kolkies** [comment]
Close this pr and create a new one and then it works I think haha

File: app/config/locale/ua.php [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<?php
+
+return [
+    'settings.inspire' => '"The art of being wise is the art of knowing what to overlook."', // This is the line printed in the homepage and console 'view-source'
+    'settings.locale' => 'ua',
+    'settings.direction' => 'rtl',
```
Comment (lex111) [inline app/config/locale/ua.php]: :warning: 

```suggestion
    'settings.direction' => 'ltr',
```

Comment (eldadfux) [reply app/config/locale/ua.php]: I will add a fix on my branch, thanks @lex111

**Comment by eldadfux** [comment]
@daniloff200 thanks for the PR, I have fixed the locale direction to LTR on my branch. Please reach out on Gitter if you want me to send you some cool Appwrite stickers and magnets! 🥳

**Pull request merged**